content/news/031/index.md

```diff
@@ -608,40 +617,109 @@ current and planned functionality.
 [Bevy]: https://github.com/bevyengine/bevy
 [@nikl_me]: https://twitter.com/nikl_me
 
-## Popular Workgroup Issues in Github
+## Discussions
 
-<!-- Up to 10 links to interesting issues -->
+<!-- Links to handpicked reddit/twitter/urlo/etc threads that provide
+useful information -->
+
+- [/r/bevy](https://www.reddit.com/r/bevy/):
+  - ["Recommended Bevy/ECS Project Structure/Conventions"]
+- [/r/rust_gamedev](https://www.reddit.com/r/rust_gamedev/):
+  - ["Which is a better game engine to start with as a beginner"]
+- [gamedev wg](https://github.com/rust-gamedev/wg)
```

Comment:
I did it for this one since it seemed to make sense for the discussions that are happening. Maybe in the future if I manage to get meetings running we can do more issue-based work :smile: